### ISSUE TITLE
Sync not working

### ISSUE THREAD

**Issue by asdffdsaasdffdsasdaer** [thomaschampagne/elevate#775]
### Step 1: Are you in the right place?

  * Your problem may have been already covered in the FAQ. Did you check https://github.com/thomaschampagne/elevate/wiki/Frequently-Asked-Questions?
  * Your problem or feature request may already exists in project issues section. Did you check https://github.com/thomaschampagne/elevate/issues?

### Step 2: Describe your environment

  * Plugin version: 6.9.2 Stable
  * Chrome/Opera/Chromium version: Version 72.0.3626.109 (Official Build) (64-bit)
  * OS version: MacOS 10.12.6
  
### Step 3: Describe the problem:
Sync not working - "Syncronisation stopped. Don't worry"
#### Steps to reproduce:

  1. Clear existing data
  2. Reinstalled plug-in
  3. Select Sync all activities OR clear and re-sync activities
  
#### Actual Behavior:

  * What happened? - Activites not synced
"Synchronization stopped. Don't worry.
The sync had to pause due to Strava servers errors or potential overload. To avoid this, please relaunch sync in few hours. It will resume where it stopped. Already synced activities will not be calculated."
  * Do you have javascript errors logs in browser console?

  > Press F12 to see developer console and javascript logs

  Failed to load resource: the server responded with a status of 403 ()
medium.jpg:1
Failed to load resource: the server responded with a status of 403 ()
medium.png:1
Failed to load resource: the server responded with a status of 404 ()
2jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23 A parser-blocking, cross site (i.e. different eTLD+1) script, https://www.google.com/uds/?file=gdata&v=1.x, is invoked via document.write. The network request for this script MAY be blocked by the browser in this or a future page load due to poor network connectivity. If blocked in this page load, it will be confirmed in a subsequent console message. See https://www.chromestatus.com/feature/5718547946799104 for more details.
google.loader.f @ jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23
2jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23 A parser-blocking, cross site (i.e. different eTLD+1) script, https://www.google.com/uds/api/gdata/1.10/208a907929e7936990d2bf48205aa8d8/core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js, is invoked via document.write. The network request for this script MAY be blocked by the browser in this or a future page load due to poor network connectivity. If blocked in this page load, it will be confirmed in a subsequent console message. See https://www.chromestatus.com/feature/5718547946799104 for more details.
google.loader.f @ jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23
medium.png:1 Failed to load resource: the server responded with a status of 404 ()
chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73 Skip handlePluginInstallOrUpgrade since we are on a sync
chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73 Sync Popup. Skip handleOnFlyActivitiesSync()
dashboard:571 Uncaught TypeError: Failed to execute 'unobserve' on 'IntersectionObserver': parameter 1 is not of type 'Element'.
    at ImpressionTracker.handleDomElementRemoved (eval at e.exports (global-dabab670fa6f4109428d.js:1), <anonymous>:571:29)
    at Mb.j (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
    at ImpressionTracker.walkNodeTree (eval at e.exports (global-dabab670fa6f4109428d.js:1), <anonymous>:489:5)
    at ImpressionTracker.Mb.j [as walkNodeTree] (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
    at ImpressionTracker.walkNodeTree (eval at e.exports (global-dabab670fa6f4109428d.js:1), <anonymous>:492:10)
    at ImpressionTracker.Mb.j [as walkNodeTree] (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
    at ImpressionTracker.walkNodeTree (eval at e.exports (global-dabab670fa6f4109428d.js:1), <anonymous>:492:10)
    at ImpressionTracker.Mb.j [as walkNodeTree] (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
    at ImpressionTracker.walkNodeTree (eval at e.exports (global-dabab670fa6f4109428d.js:1), <anonymous>:492:10)
    at ImpressionTracker.Mb.j [as walkNodeTree] (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73 PositionError
(anonymous) @ chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73
chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73 [PagesRidden] Resolving with 20 activities found
chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73 Activities fetched in group [1 => 1]: 20
/activities/2164356524/streams?stream_types[]=watts_calc&stream_types[]=watts&stream_types[]=velocity_smooth&stream_types[]=time&stream_types[]=distance&stream_types[]=cadence&stream_types[]=heartrate&stream_types[]=grade_smooth&stream_types[]=altitude&stream_types[]=latlng&stream_types[]=grade_adjusted_speed:1 Failed to load resource: the server responded with a status of 429 ()
chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73 Sync error Object
(anonymous) @ chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73
chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73 Object
(anonymous) @ chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73
chrome-extension://dhiaggccakkgdfcadnklkbljcgicpckn/core/boot.bundle.js:73 Commited:  Object
  
#### Expected Behavior:

  * What did you expect to happen?
Expected the sync to process my approximately 1760 strava activites
  
#### Media attachement:

>You can upload/attach any images to this ticket.

#### Providing your Elevate synced athlete activities can be useful to solve the problem

> To export your activities click on "sync button" (top right and corner), then "Backup activities". You can zip the file and upload it here.

**Comment by thomaschampagne** [comment]
Indeed as written "please relaunch sync in few hours"

**Comment by MCO7** [comment]
Hello, same error here Yesterday. After a relauch this morning, same error. (Everytime at the same point "Activity Group 7 - 70%"

Total activities found 964
Total activities processed 120
Extension local storage occupation: 0.4MB

**Comment by thomaschampagne** [comment]
It's an expected behaviour. Sync works but with limitations. You have to wait more between syncs don't know how many yet

**Comment by MCO7** [comment]
> It's an expected behaviour. Sync works but with limitations. You have to wait more between syncs don't know how many yet

Got it, thanks for the answer. I'll let you know

**Comment by MCO7** [comment]
Same Today... ALways stuck at this point :+1: 
Global synchronisation progress 12%
Activity group 7 ➝ Fetching streams... 75%
 
16Unchecked runtime.lastError: The message port closed before a response was received.
VM936:571 Uncaught TypeError: Failed to execute 'unobserve' on 'IntersectionObserver': parameter 1 is not of type 'Element'.
    at ImpressionTracker.handleDomElementRemoved (eval at e.exports (addScript.js:20), <anonymous>:571:29)
    at Mb.j (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
    at ImpressionTracker.walkNodeTree (eval at e.exports (addScript.js:20), <anonymous>:489:5)
    at ImpressionTracker.Mb.j [as walkNodeTree] (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
    at ImpressionTracker.walkNodeTree (eval at e.exports (addScript.js:20), <anonymous>:492:10)
    at ImpressionTracker.Mb.j [as walkNodeTree] (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
    at ImpressionTracker.walkNodeTree (eval at e.exports (addScript.js:20), <anonymous>:492:10)
    at ImpressionTracker.Mb.j [as walkNodeTree] (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
    at ImpressionTracker.walkNodeTree (eval at e.exports (addScript.js:20), <anonymous>:492:10)
    at ImpressionTracker.Mb.j [as walkNodeTree] (core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8)
ImpressionTracker.handleDomElementRemoved @ VM936:571
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.walkNodeTree @ VM936:489
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.walkNodeTree @ VM936:492
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.walkNodeTree @ VM936:492
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.walkNodeTree @ VM936:492
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.walkNodeTree @ VM936:492
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.walkNodeTree @ VM936:492
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.walkNodeTree @ VM936:492
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.walkNodeTree @ VM936:492
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.walkNodeTree @ VM936:492
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
ImpressionTracker.handleDomMutations @ VM936:471
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
childList (async)
(anonymous) @ VM931:3
domManip @ VM931:3
before @ VM931:3
e.modify @ boot.bundle.js:73
t.handleMenu @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.run @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
e.emitStartCoreEvent @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
e.loaded @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
load (async)
e.writeScript @ content.bundle.js:1
e.require @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
Promise.then (async)
e.start @ content.bundle.js:1
231 @ content.bundle.js:1
n @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
boot.bundle.js:73 PositionError {code: 1, message: "User denied Geolocation"}
(anonymous) @ boot.bundle.js:73
dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7 GET https://www.strava.com/activities/2162408483/streams?stream_types[]=watts_calc&stream_types[]=watts&stream_types[]=velocity_smooth&stream_types[]=time&stream_types[]=distance&stream_types[]=cadence&stream_types[]=heartrate&stream_types[]=grade_smooth&stream_types[]=altitude&stream_types[]=latlng&stream_types[]=grade_adjusted_speed 404
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.fetchStreamByActivityId @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getSyncedActivitiesFromLocal @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
setTimeout (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
t.fetchWithStream @ boot.bundle.js:73
t.fetchAndComputeGroupOfPages @ boot.bundle.js:73
t.computeActivitiesByGroupsOfPages @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getLastSyncDateFromLocal @ boot.bundle.js:73
t.sync @ boot.bundle.js:73
e.sync @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
each @ VM931:2
each @ VM931:2
e.modify @ boot.bundle.js:73
t.handleActivitiesSyncFromOutside @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.run @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
e.emitStartCoreEvent @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
e.loaded @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
load (async)
e.writeScript @ content.bundle.js:1
e.require @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
Promise.then (async)
e.start @ content.bundle.js:1
231 @ content.bundle.js:1
n @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
(anonymous) @ content.bundle.js:1
dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7 GET https://www.strava.com/activities/1985000816/streams?stream_types[]=watts_calc&stream_types[]=watts&stream_types[]=velocity_smooth&stream_types[]=time&stream_types[]=distance&stream_types[]=cadence&stream_types[]=heartrate&stream_types[]=grade_smooth&stream_types[]=altitude&stream_types[]=latlng&stream_types[]=grade_adjusted_speed 404
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.fetchStreamByActivityId @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getSyncedActivitiesFromLocal @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
setTimeout (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
t.fetchWithStream @ boot.bundle.js:73
t.fetchAndComputeGroupOfPages @ boot.bundle.js:73
t.computeActivitiesByGroupsOfPages @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getSyncedActivitiesFromLocal @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
S.i.notify @ boot.bundle.js:66
n.onmessage @ boot.bundle.js:73
dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7 GET https://www.strava.com/activities/1717461212/streams?stream_types[]=watts_calc&stream_types[]=watts&stream_types[]=velocity_smooth&stream_types[]=time&stream_types[]=distance&stream_types[]=cadence&stream_types[]=heartrate&stream_types[]=grade_smooth&stream_types[]=altitude&stream_types[]=latlng&stream_types[]=grade_adjusted_speed 500
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.fetchStreamByActivityId @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getSyncedActivitiesFromLocal @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
setTimeout (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
t.fetchWithStream @ boot.bundle.js:73
t.fetchAndComputeGroupOfPages @ boot.bundle.js:73
t.computeActivitiesByGroupsOfPages @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getSyncedActivitiesFromLocal @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
S.i.notify @ boot.bundle.js:66
n.onmessage @ boot.bundle.js:73
boot.bundle.js:73 Sync error {streamFailure: true, activityId: 1717461212, statusCode: 500, statusText: "error"}
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
when @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o.runAfter @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
B @ boot.bundle.js:66
S.i.reject @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getSyncedActivitiesFromLocal @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
setTimeout (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
t.fetchWithStream @ boot.bundle.js:73
t.fetchAndComputeGroupOfPages @ boot.bundle.js:73
t.computeActivitiesByGroupsOfPages @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getSyncedActivitiesFromLocal @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
S.i.notify @ boot.bundle.js:66
n.onmessage @ boot.bundle.js:73
boot.bundle.js:73 {streamFailure: true, activityId: 1717461212, statusCode: 500, statusText: "error"}
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
when @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o.runAfter @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
B @ boot.bundle.js:66
S.i.reject @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getSyncedActivitiesFromLocal @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
setTimeout (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
(anonymous) @ boot.bundle.js:73
(anonymous) @ VM931:2
j @ VM931:2
fireWith @ VM931:2
x @ VM931:4
(anonymous) @ VM931:4
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
load (async)
f @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
o @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
XMLHttpRequest.send (async)
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
send @ VM931:4
ajax @ VM931:4
t.httpPageGet @ boot.bundle.js:73
t.fetchRawActivitiesRecursive @ boot.bundle.js:73
t.fetchWithStream @ boot.bundle.js:73
t.fetchAndComputeGroupOfPages @ boot.bundle.js:73
t.computeActivitiesByGroupsOfPages @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
Promise.then (async)
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
o @ boot.bundle.js:66
S.i.resolve @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:73
Promise.then (async)
t.getSyncedActivitiesFromLocal @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:73
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
T.r.promiseDispatch @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
c @ boot.bundle.js:66
l @ boot.bundle.js:66
(anonymous) @ VM939:2
u @ VM939:2
n @ VM939:2
nrWrapper @ dashboard?elevateSync=true&fastSync=false&forceSync=true&sourceTabId=4:7
postMessage (async)
r @ VM939:2
a.setImmediate @ VM939:2
Mb.j @ core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js:8
o @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
(anonymous) @ boot.bundle.js:66
S.i.notify @ boot.bundle.js:66
n.onmessage @ boot.bundle.js:73

**Comment by MCO7** [comment]
Salut, je suis désolé de te contacter par là. Je ne peux toujours pas sync mes données ça m'embête vraiment car j'aime beaucoup les fonctionnalités apportées par ce plugin.
Il y a t il un moyen de ne récupérer que les 12 derniers mois par exemple ?
Merci.
